src/components/MarkdownProvider/components/ColorPalette.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { readableColor } from 'polished';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledColorHex = styled.figcaption`
+  margin-left: auto;
+  font-size: ${p => p.theme.fontSizes.sm};
+`;
+
+const StyledColorPalette = styled.div`
+  margin-top: ${p => p.theme.space.md};
+  margin-bottom: ${p => p.theme.space.xl};
+`;
+
+const StyledColorSwatch = styled.figure<{ color: string }>`
+  display: flex;
+  align-items: center;
+  background-color: ${p => p.color};
+  padding: ${p => p.theme.space.xs} ${p => p.theme.space.sm};
+  color: ${p => readableColor(p.color, p.theme.colors.foreground, p.theme.colors.background)};
+
+  &:first-child {
+    border-top-left-radius: ${p => p.theme.space.sm};
+    border-top-right-radius: ${p => p.theme.space.sm};
+  }
+
+  &:last-child {
+    border-bottom-left-radius: ${p => p.theme.space.sm};
+    border-bottom-right-radius: ${p => p.theme.space.sm};
+  }
+`;
+
+const StyledColorTitle = styled.b`
+  white-space: nowrap;
+  font-size: ${p => p.theme.fontSizes.md};
+  font-weight: ${p => p.theme.fontWeights.semibold};
+`;
+
+const StyledRow = styled(Row)`
+  & + & {
+    margin-top: ${p => p.theme.space.md};
```

Comment:
Maybe this gets applied to the `Col`s to stop [this from happening on mobile](https://share.getcloudapp.com/RBuWGPxr)?

Reply:
doh, of course. Thanks for spotting this.